app/views/ExposureHistory/NextSteps.tsx

```diff
@@ -82,6 +76,7 @@ const styles = StyleSheet.create({
   },
   button: {
     ...Buttons.largeBlue,
+    borderWidth: 1,
```

Comment:
does this have an explicit color associated with the button spread? This border width seems like it could add an unwanted color as is?

Reply:
yup! this should be removed.